patches/openssl/0002-Add-configuration-file-for-Managarm-targets.patch

```diff
@@ -0,0 +1,68 @@
+From d3811a0350dcc9d53d919b5c7b69b92173f1f48e Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Tue, 1 Jun 2021 22:03:10 +0200
+Subject: [PATCH 2/2] Add configuration file for Managarm targets. Initial work
```

Comment:
this commit message could be formed better

Reply:
Hmmm. I guess I can make the second sentence look more like a proper sentence if that is what you mean, otherwise I’m not really sure what you want me to improve.

Reply:
> openssl: add config file for managarm target

https://chris.beams.io/posts/git-commit/#limit-50
https://chris.beams.io/posts/git-commit/#wrap-72

your editor should assist you with these two basic rules, at least, see `ft=gitcommit` in vim.

Reply:
Well, the first line does not fit into 50 chars; I guess that is what Arsen means. We're not really adding a file anymore; we can also just drop "Initial work".